src/components/Logo/Logo.styles.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import LogoRed from '../../images/logoRed.png';
+import LogoBlue from '../../images/logoBlue.png';
+
+export const LogoWrapper = styled.div`
+  background-image: url(${LogoRed});
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: ${(props) => (props.width ? `${props.width}rem` : '18.8rem')};
+  height: ${(props) => (props.height ? `${props.height}rem` : '8.2rem')};
+  &:hover {
+    background-image: url(${LogoBlue});
+  }
+`;
```

Comment:
Have two options for different colour logs, hover state should be the opposite color for each.

Reply:
I have added 'color' prop and on hover it shows the opposite color.